protocols/system/netlink/netlink.go

```diff
@@ -0,0 +1,224 @@
+package netlink
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/bio-routing/bio-rd/config"
+	"github.com/bio-routing/bio-rd/net"
+	"github.com/bio-routing/bio-rd/route"
+	"github.com/bio-routing/bio-rd/routingtable"
+	rt "github.com/bio-routing/bio-rd/routingtable"
+	"github.com/bio-routing/bio-rd/routingtable/locRIB"
+	log "github.com/sirupsen/logrus"
+	"github.com/vishvananda/netlink"
+
+	bnet "github.com/bio-routing/bio-rd/net"
+)
+
+type NetlinkServer struct {
```

Comment:
Isn't NetlinkClient a better name? Or is it a Server in the terminology in the netlink API?